compaction_picker.go

```diff
@@ -1287,6 +1293,10 @@ func (p *compactionPickerByScore) pickAuto(env compactionEnv) (pc *pickedCompact
 		}
 	}
 
+	if pc := p.pickTombstoneDensityCompaction(env); pc != nil {
```

Comment:
Probably worth adding a comment here and distinguishing this from elision only compactions (below), which also talk about tombstones. The difference being primarily that tombstone density compactions prioritize tombstones in all levels, while elision only ones are specific to tombstones/obsolete keys retained by snapshots.